README.md

```diff
@@ -1,3 +1,5 @@
+![build status](https://circleci.com/gh/docker/cli.svg?style=shield&circle-token=3de88f141d4323ea4971a870ef706abc6740ef27)
```

Comment:
Is this a special RO access token? (https://circleci.com/docs/1.0/status-badges/#access-control)

I think we can omit it since the repo is open source? I'm able to view it from an incognito browser.

Should we also make this image a link to the build?